apps/jsii-docgen/test/docgen/view/__snapshots__/markdown.test.ts.snap

```diff
@@ -21,49 +21,49 @@ new GreeterBucket(props?: BucketProps)
 
 | **Name** | **Type** | **Description** |
 | --- | --- | --- |
-| <code><a href=\\"#custom-construct-library.GreeterBucket.Initializer.parameter.props\\">props</a></code> | <code><a href=\\"#custom-@aws-cdk/aws-s3.BucketProps\\">BucketProps</a></code> | *No description.* |
+| <code><a href=\\"/packages/construct-library/v/0.0.0/api/GreeterBucket.Initializer.parameter.props?lang=python\\">props</a></code> | <code><a href=\\"/packages/@aws-cdk/aws-s3/v/99.99.99/api/BucketProps?lang=python\\">BucketProps</a></code> | *No description.* |
```

Comment:
hmm, these changes dont seem intentional. Im not sure why there is all this version 0.0.0 and `lang=python` updates. @Chriscbr or @MarkMcCulloh any ideas?

Reply:
Not sure off the top of my head - let's try to just remove it from the branch for now